### ISSUE TITLE
Possibility of single key build command?

### ISSUE THREAD

**Issue by MythicAngel** [Microsoft/vscode#17880]
- VSCode Version: 1.8.1
- OS Version: Windows 8 (64 bit)

**Query:**
I need this badly for working in typescript. I need to pause working, build with ctrl and space key in one hand and B in another, Again mouse click on browser, click refresh to load new JS. Therefore, a single build command would definitely help to reduce this task.
Visual studio provides a simple build key (F6), can we provide the same?

In my opinion, build is a continuous and one of the most vital parts of source code IDE. Therefore, thinking of such solution. I have also checked keyboard shortcut reference and confirm that it is not yet booked.

**Comment by dbaeumer** [comment]
@MythicAngel I don't understand what exactly you are proposing. Is Ctrl+Shift+B not a single build command. If you are unhappy with the keyboard short cut you can always rebind this to something else via File > Preferences > Keyboard Shortcuts

**Comment by MythicAngel** [comment]
@dbaeumer 
My opinion comes from the usage of VisualStudio or AndroidStudio, where write code in IDE, then press the build button and test somewhere else (e.g. Console application in CMD, android device). VSCode has lots of similarity, where the built code is used in another system. Notable example would be typescript, where we write code in VSCode, then build it and use it in Javascript.

VSCode provides build option, but the shortcut (_ctrl+shift+b_) is not visible from menu entries. Therefore, I suggested to provide an entry in menu saying "Build (F6)" which will trigger "workbench.action.tasks.build" task. 
I checked keyboard references and found that no key assigned to F6 key, thus we can mimic this.

Hope it clarifies your questions.
